cmd/node-termination-handler.go

```diff
@@ -113,6 +113,13 @@ func main() {
 		nthConfig.Print()
 		log.Fatal().Msgf("Unable to find the AWS region to process queue events.")
 	}
+
+	recorder, err := observability.InitK8sEventRecorder(nthConfig.EmitKubernetesEvents, nthConfig.NodeName, nodeMetadata, nthConfig.KubernetesEventsExtraAnnotations)
+	if err != nil {
+		nthConfig.Print()
+		log.Fatal().Err(err).Msg("Unable to create Kubernetes event recorder,")
+	}
```

Comment:
That's where I had it initially, but then I had to move it down when I added the IMDS annotations stuff since I need `nodeMetadata` to be passed in. Happy to move it right after `nodeMetadata := imds.GetNodeMetadata()` at 105 if you prefer.

Reply:
ah I see, it's fine as is